### PR TITLE
 Add ability to supply existing workspace not managed by the runner

### PR DESCRIPTION
**Pull request by odinuv**
Part of https://keboola.atlassian.net/browse/KAB-257
Depends on https://github.com/keboola/docker-bundle/pull/763
Depends on https://github.com/keboola/platform-libraries/pull/315

Vyzkoušený tady https://connection.east-us-2.azure.keboola-testing.com/admin/projects/18/queue/3856771

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,11 +23,12 @@
         "aws/aws-sdk-php": "^3.258",
         "keboola/api-error-control": "^4.2",
         "keboola/configuration-variables-resolver": "^5.1",
-        "keboola/dockerbundle": "dev-main",
+        "keboola/dockerbundle": "dev-odin-KAB-257-2",
```
Comment (odinuv) [inline composer.json:26]: vrátit do původního stavu

File: src/JobDefinitionParser.php [hunk 2]
```diff
@@ -93,7 +93,7 @@ public function parseConfig(
                 (bool) $row['isDisabled'],
                 $branchType,
             ),
-            $config['rows'],
+            (array) $config['rows'],
```
Comment (pepamartinec) [inline src/JobDefinitionParser.php:96]: Zas nejaky update PHPStan?

Comment (odinuv) [reply src/JobDefinitionParser.php:96]: jj

**Review by pepamartinec — APPROVED** [review]
lgtm

**Pull request merged**